app/models/exchange_rate_currency_rate.rb

```diff
@@ -111,14 +111,6 @@ def by_month_and_year(month, year)
       )
       .order(Sequel.desc(:applicable_date))
     end
-
-    # This query removes country/currency pairings where the country/currency
-    # is expiring before the end of the rate
-    def without_expired_countries
```

Comment:
Wont removing this cause an issue for expired countries?

Reply:
Its a duplication

Reply:
ahh ok can see i added this ages ago but was trying to remember why it was added or what it caught

Reply:
Just in case I added an extra test to the method that was calling it